engine/samples/input/main.cpp

```diff
@@ -0,0 +1,198 @@
+#include <cubos/core/data/debug_serializer.hpp>
+#include <cubos/core/settings.hpp>
+
+#include <cubos/engine/assets/plugin.hpp>
+#include <cubos/engine/input/bindings.hpp>
+#include <cubos/engine/input/plugin.hpp>
+
+using cubos::core::Settings;
+using cubos::core::data::Debug;
+using cubos::core::ecs::Read;
+using cubos::core::ecs::Write;
+using namespace cubos::engine;
+
+static const Asset<InputBindings> bindingsAsset = AnyAsset("bf49ba61-5103-41bc-92e0-8a442d7842c3");
+int showcase = 0;
+bool nextPressed = false;
+bool explained = false;
```

Comment:
I know this is just a sample but since it is a sample we should show good practices: this should be a resource.
Add something like
```suggestion
struct State
{
    int showcase = 0;
    bool nextPressed = false;
    bool explained = false;
};
```